tests/js-tests/src/CocosDenshionTest/CocosDenshionTest.js

```diff
@@ -231,7 +231,11 @@ var soundId = null;
 
 var playMusic = function () {
     cc.log("play background music");
-    audioEngine.playMusic(MUSIC_FILE, false);
+    var musicFile = MUSIC_FILE;
+    if (cc.sys.os == cc.sys.OS_ANDROID) {
+        musicFile = "res/"+musicFile;
+    }
+    audioEngine.playMusic(musicFile, false);
```

Comment:
@minggo Please refer https://github.com/cocos2d/cocos2d-x/blob/v3/tests/js-tests/main.js#L118, these search path will effect in C/C++ layer, but it won't effect in Java layer